tool_run_scripts/stats.py

```diff
@@ -80,11 +80,10 @@ def print_fails(self, fail_count=5, output=None, verbose=True):
 
     def get_fail_count(self):
         success_runs = len(self.stats.get("output.success", []))
-        timed_out_runs = self.stats.get("program_timeouts", 0)
         program_runs = self.stats.get("program_runs", 0)
         ignored_outputs = len(self.stats.get("outputignore_success", []))
         ignored_outputs += len(self.stats.get("outputignore_fail", []))
-        return (program_runs - ignored_outputs) - (success_runs + timed_out_runs)
+        return program_runs - ignored_outputs - success_runs
```

Comment:
This is another minor fix that I've included.

The `timed_out_runs` stat includes ignored tests so we can get a negative fail count here. Also timing out should be considered a failure if we're not ignoring the test (in my opinion).

Reply:
hmm yeah this was a temprorary fix to try to get anvill CI passing with the timeouts we have but I agree in principle